### PR TITLE
[Exaqute] Extending wind generator to MPI 

### PR DESCRIPTION
**Pull request by marcnunezc**
**Description**
WIP

**Changelog**
Please summarize the changes in one list to generate the changelog:
E.g.
- Added feature X to Y
- Added Foo Application
- Fixed X (#XXXX Reference to issue if apply) 
- etc...

### CODE REVIEW

File: applications/ExaquteSandboxApplication/python_scripts/WindGenerator/onthefly_mpi_kratos_inlet_process.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            self.blend_region, self.mappers = self.Create3DMappers(self.wind, self.blend_region)
+
+    def ApplyRamp(self):
+        time = self.mpi_model_part.ProcessInfo[KratosMultiphysics.TIME]
+        if time < self.ramp_time:
+            scal = time / self.ramp_time
+            for node in self.inlet_nodes:
+                for var, _ in self.mappers:
+                    vel = node.GetSolutionStepValue(var)
+                    node.SetSolutionStepValue(var, scal * vel)
```
Comment (riccardotosi) [inline applications/ExaquteSandboxApplication/python_scripts/WindGenerator/onthefly_mpi_kratos_inlet_process.py:65]: Are these two methods required to be defined here? Can't we use the base class methods?

Comment (marcnunezc) [reply applications/ExaquteSandboxApplication/python_scripts/WindGenerator/onthefly_mpi_kratos_inlet_process.py:65]: We should update the methods to depend on the main model part, note that TIME and DELTA_TIME is read from the `self.mpi_model_part`, which is not the `self.inlet_model_part` that is used to compute the wind. 

So do something like this in the base class:

  ```
  def UpdateInletPosition(self, this_model_part):
        dt = this_model_part.ProcessInfo[KratosMultiphysics.DELTA_TIME]
        self.inlet_position += dt * self.mean_profile.bulk_wind_speed

        #If the end of the time block is reached, generate a new wind block and reset map
        if self.inlet_position >= self.x0 + self.lx:
            self.x0 += self.lx
            self.blend_region, self.mappers = self.Create3DMappers(self.wind, self.blend_region)
```

Either this or have `dt` as input parameter

Comment (riccardotosi) [reply applications/ExaquteSandboxApplication/python_scripts/WindGenerator/onthefly_mpi_kratos_inlet_process.py:65]: Ok I did not catch the usage of `mpi_model_part`. To me, it's cleaner to do it as you suggest in this comment. No?

**Comment by riccardotosi** [comment]
We can probably close this, since we are working in #8991.

**Pull request closed without merging**